controller/commentsController.js

```diff
@@ -405,3 +405,40 @@ exports.getCommentVotes = async (req, res, next) => {
     );
   }
 };
+exports.getSingleComment = async (req, res, next) => {
+  const { refId } = req.query;
```

Comment:
This query parameter does not exist on this endpoint and shouldn't be accounted for. Could you please kindly remove this refId query param.